src/browser/api/application.js

```diff
@@ -524,6 +535,21 @@ Application.run = function(identity, configUrl = '' /*, callback , errorCallback
             type: 'connected',
             uuid
         });
+
+        const parentConfigUrl = coreState.getConfigUrlByUuid(uuid);
+
+        rvmBus.registerLicenseInfo({
+            licenseKey: mainWindowOpts.licenseKey,
```

Comment:
`licenseKey` isn't at the options level, it's at the top app config level i think